### PR TITLE
applying chris's requirements syntax to workday-reports

### PR DESCRIPTION
**Pull request by mozafrank**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,9 +3,8 @@
 from setuptools import setup, find_packages
 from pip._internal.req import parse_requirements
```
Comment (cvalaas) [inline setup.py]: remove this line, too, since it's the first one to break

**Review by cvalaas — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (cvalaas) [reply setup.py]: the `from pip._internal.req import parse_requirements` i mean, if it isn't clear

Comment (mozafrank) [reply setup.py]: ahh my bad, fixed!

**Review by cvalaas — APPROVED** [review]
lgtm

**Pull request merged**